docs/system_overview/configuration.md

```diff
@@ -645,6 +645,40 @@ This is the main iRODS configuration file defining the iRODS environment. Any ch
     // The Diffie-Hellman parameter file location.
     "irods_ssl_dh_params_file": "",
 
+    // (Optional)
+    // The number of seconds between TCP keep-alive probes. For more details, see:
+    //  - man tcp
+    //  - https://tldp.org/HOWTO/TCP-Keepalive-HOWTO/usingkeepalive.html
+    //
+    // This should be a non-negative integer. If this option is not set or is set to an invalid value, the socket
+    // option for TCP_KEEPINTVL will remain unset. This means that the value of tcp_keepalive_intvl in use for the
+    // kernel configuration will be used for iRODS sockets using TCP keepalive.
+    "tcp_keepalive_intvl_in_seconds": 75,
+
+    // (Optional)
+    // The maximum number of TCP keep-alive probes to send before giving up and killing the connection if no
+    // response is obtained from the other end. For more details, see:
+    //  - man tcp
+    //  - https://tldp.org/HOWTO/TCP-Keepalive-HOWTO/usingkeepalive.html
+    //
+    // This should be a non-negative integer. If this option is not set or is set to an invalid value, the socket
+    // option for TCP_KEEPCNT will remain unset. This means that the value of tcp_keepalive_probes in use for the
+    // kernel configuration will be used for iRODS sockets using TCP keepalive.
+    "tcp_keepalive_probes": 9,
+
+    // (Optional)
+    // The number of seconds a connection needs to be idle before TCP begins sending out keep-alive probes.
+    // The default value is 7200 seconds (2 hours). An idle connection is terminated after approximately an
+    // additional 11 minutes (9 probes an interval of 75 seconds apart) when keep-alive is enabled. For more
```

Comment:
Is `9 probes an` supposed to be `9 probes at an`?

Reply:
This is the wording in the TCP man page. I'm willing to change the wording if that helps since I've already made a couple of edits for brevity.

Reply:
If it can be made less confusing, then let's change it so others don't misunderstand it.

Reply:
Done